Makefile

```diff
@@ -359,12 +359,11 @@ operator-lint: gowork ## Runs operator-lint
 	go vet -vettool=$(LOCALBIN)/operator-lint ./... ./api/...
 
 # Used for webhook testing
-# Please ensure the neutron-controller-manager deployment and
-# webhook definitions are removed from the csv before running
-# this. Also, cleanup the webhook configuration for local testing
-# before deplying with olm again.
-# $oc delete -n openstack validatingwebhookconfiguration/vneutronapi.kb.io
-# $oc delete -n openstack mutatingwebhookconfiguration/mneutronapi.kb.io
+# The configure_local_webhooks.sh script below will remove any OLM webhooks
+# for the operator and also scale its deployment replicas down to 0 so that
+# the operator can run locally.
+# Make sure to cleanup the webhook configuration for local testing by running
+# ./hack/clean_local_webhook.sh before deplying with OLM again.
```

Comment:
may be can do consider running clean_local_webhook.sh automatically on some signals ctrl + C?

Reply:
That's possible.  Maybe in a follow-up investigation?

Reply:
follow up should be fine.